source/frontend/src/features/mapSideBar/acquisition/tabs/expropriation/form1/ExpropriationForm1.tsx

```diff
@@ -0,0 +1,110 @@
+import { Formik, FormikHelpers, FormikProps } from 'formik';
+import React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+
+import { Button } from '@/components/common/buttons';
+import { Input } from '@/components/common/form';
+import { ContactInputContainer } from '@/components/common/form/ContactInput/ContactInputContainer';
+import ContactInputView from '@/components/common/form/ContactInput/ContactInputView';
+import FormItem from '@/components/common/form/FormItem';
+import { SectionField } from '@/components/common/Section/SectionField';
+import { RestrictContactType } from '@/components/contact/ContactManagerView/ContactFilterComponent/ContactFilterComponent';
+import FilePropertiesTable from '@/components/filePropertiesTable/FilePropertiesTable';
+import { Api_AcquisitionFile } from '@/models/api/AcquisitionFile';
+import { Api_PropertyFile } from '@/models/api/PropertyFile';
+
+import { ExpropriationForm1Model } from '../models';
+import { ExpropriationForm1YupSchema } from './ExpropriationForm1YupSchema';
+
+export interface IExpropriationForm1Props {
+  acquisitionFile: Api_AcquisitionFile;
+  onGenerate: (values: ExpropriationForm1Model) => Promise<void>;
+}
+
+export const ExpropriationForm1: React.FC<IExpropriationForm1Props> = ({
+  acquisitionFile,
+  onGenerate,
+}) => {
+  const onSubmit = async (
+    values: ExpropriationForm1Model,
+    formikHelpers: FormikHelpers<ExpropriationForm1Model>,
+  ) => {
+    try {
+      await onGenerate(values);
+    } finally {
+      formikHelpers?.setSubmitting(false);
+    }
+  };
+
+  const onCancelClick = (formikProps: FormikProps<ExpropriationForm1Model>) => {
+    formikProps.resetForm();
+  };
+
+  const onGenerateClick = (formikProps: FormikProps<ExpropriationForm1Model>) => {
+    formikProps.setSubmitting(true);
+    formikProps.submitForm();
+  };
+
+  return (
+    <Formik<ExpropriationForm1Model>
+      enableReinitialize
+      initialValues={new ExpropriationForm1Model()}
+      validationSchema={ExpropriationForm1YupSchema}
+      onSubmit={onSubmit}
+    >
+      {formikProps => (
+        <React.Fragment>
```

Comment:
hmm, do we need unsaved changes warnings?

Reply:
Form 1, 5, 9 don't save anything so not sure about that warning. I do agree that if the user navigates away they loose their "in-memory" work. Since there's no clear AC calling for unsaved changes warning I'd wait for feedback on it